Notes/NoteViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  NoteViewController.swift
+//  Notes
+//
+//  Created by Kathy Nguyen on 2/22/21.
+//
+
+import UIKit
+
+
+class NoteViewController: UIViewController, UITextFieldDelegate {
+    @IBOutlet var titleLabel : UILabel!
+    @IBOutlet var editableTitleLabel: UITextField!
+    @IBOutlet var noteField: UITextView!
+    
+    // need to give these default values or will get "no initializers" error
+    public var noteTitle: String = ""
+    public var note: String = ""
+    public var completion: ((String, String) -> Void)?
```

Comment:
Why do you prefer callbacks over delegation?

Reply:
Same reasons as [this](https://github.com/kathyn262/swiftnotesapp/pull/1#discussion_r587714318) 

Also ended up changing this to delegation